dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/support/AbortPolicyWithReport.java

```diff
@@ -158,6 +158,7 @@ private void dumpJStack() {
         }
         // To avoid multiple dump, check again
         if (System.currentTimeMillis() - lastPrintTime < TEN_MINUTES_MILLS) {
+            guard.release();
```

Comment:
Maybe we can move 
```
        if (System.currentTimeMillis() - lastPrintTime < TEN_MINUTES_MILLS) {
            guard.release();
            return;
        }

        ExecutorService pool = Executors.newSingleThreadExecutor();
```
into try-finally block

Reply:
sorry,I didn't unstand.I haven't realized the advantages of moving these codes into try-finally block.
in this what you means? 

        if (!guard.tryAcquire()) {
            return;
        }
        
        ExecutorService pool;
        try {
            // To avoid multiple dump, check again
            if (System.currentTimeMillis() - lastPrintTime < TEN_MINUTES_MILLS) {
                guard.release();
                return;
            }
            pool = Executors.newSingleThreadExecutor();
            ...........

Reply:
guard can be released in the finally block

Reply:
done, please take another look.